app/components/Options/Options.js

```diff
@@ -32,58 +32,58 @@ export default class Options extends Component {
 
     return (
       <ul className={style.options}>
-      {groups.map((group) =>
-        <li key={group.id}>
-          <span
-            id={group.id}
```

Comment:
Span elements must never have children that are div elements, because `inline` elements should never have `block` elements as children. It's semantically passable but IDEs such as PHPStorm get confused when they try to auto-format the code, which reinforces the notion that we want to avoid this.